src/Stratis.Bitcoin/Consensus/ValidationResults/PartialValidationResult.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Stratis.Bitcoin.Primitives;
```

Comment:
Why do we need to put small similar classes into several files? 

This one and ValidationResult can be in the same file.

Reply:
I think splitting into smaller files helps reducing the amount of information you have to keep in mind, if you then need this information you can have it on a separate window, or peek at it.
Also, when many contributors are involved it makes changes less likely to conflict for instance...

Reply:
Too much of separation actually makes it less readable. If it is something very closely related, it is more readable for me to have it in the same file.